design-editor/src/pane/guide.js

```diff
@@ -264,14 +278,15 @@ class Guide {
         return $container.children('[' + INTERNAL_ID_ATTRIBUTE + ']' + selector);
     }
 
-    /**
-     * Insert Rule
-     * @param {Object} targetInfo
-     * @param {jQuery} $container
-     * @param {boolean} isRestrict
-     * @returns {*}
-     * @private
-     */
+	/**
+	 * Responsible for way in which element is inserted to document
+	 * before or after elements in the same scope
+	 * @param {Object} targetInfo - current target (first element under mouse pointer)
+	 * @param {jQuery} $container - container defined by component to be its parent
+	 * @param {boolean} isRestrict
+	 * @returns {*}
```

Comment:
I know it's not your code but we can either remove the @returns or describe what is returned

Reply:
When you use `@` symbol, you are calling somebody from GitHub using their username. If you want to avoid this, please use names of JS-Doc rules formatted as code. You can do it by surrounding text by backticks.